requirements.txt

```diff
@@ -64,9 +64,11 @@ Django==3.1.7
 django-crispy-forms==1.13.0
 django-crontab==0.7.1
 django-extensions==3.1.5
+django-model-utils-4.2.0
 django-template-maths==0.2.0
 docutils==0.16
 entrypoints==0.3
+environs==9.3.5
```

Comment:
is this correct? following the requirements I've installed environs through pip and got the following:
`raise ValueError, "No frame marked with %s." % fname`
which was solved by installing django-environ instead

Reply:
No, I should have removed this, as we don't need this library. I will correct it.